.gitignore

```diff
@@ -4,6 +4,9 @@ __pycache__/
 *$py.class
 .DS_Store
 
+# TODO File
+my_contributions.txt
+
```

Comment:
Can you remove this? Seem pretty related to your personal workflow (trying to keep entropy under control)

Reply:
Thanks @fpliger. I've made the requested changes